### ISSUE TITLE
Jumpy video when starting service playback

### ISSUE THREAD

**Issue by MrQuade** [beyonwiz-dev/enigma2#5]
Whenever a service is started by any means (CH+/CH-, Via Favourites, via EPG etc), the first 0.5s to 5s (variable) of playback is not smooth.

This momentary jumpy playback is also evident after executing a skip using the number keypad while playing any recording.

**Comment by atvcaptain** [comment]
yes is hisi hw depend and cant be fixed its allready report no solution at the moment possible.

**Comment by peterurbanec** [comment]
It looks a lot like the playback engine is trying to synchronise the video with the audio, but it does not have all the required video frames. Probably a driver thing. Broadcom SDK provided some control over the method, but from my experience the only mode that provided satisfactory results was to mute both audio and video until synch was achieved. This usually didn't take much longer than one GOP.

V2 appears to take longer than a GOP, but without seeing the driver source and getting familiar with the HiSi SDK we have to wait for the hardware vendor to find a solution. (if there is one)

**Comment by atvcaptain** [comment]
its report to hisi , we can only wait for a new sdk

**Comment by WizHQ** [comment]
Please note that this problem also happens when you skip in timeshift or media playback, if you skip 10secs at a time every skip it takes time to sync, also if you skip fast it can become unstable. Funny think is that this does not happen with these 4K files I have been testing, if you skip it is smooth and no sync issues: http://beyonwiz.com.au/sample4k/4k/4k%20demo%2060fps%20bike%20samsung%20test%20video%20for%20qled%20%282160p_30fps_H264-128kbit_AAC%29.mp4
http://beyonwiz.com.au/sample4k/4k/BSR%20SUPER%20SLIDE%20-%20The%20Royal%20Flush%20in%204K%20%282160p_30fps_H264-128kbit_AAC%29.mp4
http://beyonwiz.com.au/sample4k/4k/ok_Sony_4K_HDR_Camp.mp4

**Comment by atvcaptain** [comment]
yes all hisi e2 models show this
